docs/architecture/index.md

```diff
@@ -0,0 +1,9 @@
+# Architecture
+
+This living document serves as a high-level documentation to explain the conceptual choices and tool choices in the Gutenberg repository.
+
+- [Modularity and WordPress Packages](docs/architecture/modularity.md).
+- [Block Editor Performance](docs/architecture/performance.md).
+- What are the decision decisions behind the Data Module?
+- [Why is Puppeteer the tool of choice for end-to-end tests?](/docs/architecture/automated-testing.md)
```

Comment:
In dd33523, I added a document addressing this question.

Reply:
Thanks these are great.